FastStyleTransfer/Layers/Normalization.swift

```diff
@@ -0,0 +1,24 @@
+import TensorFlow
+
+/// Layer that applies instance normalization over a mini-batch of images
+/// Expected input layout is BxHxWxC
+/// Reference: [Instance Normalization](https://arxiv.org/abs/1607.08022)
+public struct InstanceNorm2d<Scalar: TensorFlowFloatingPoint>: Layer {
```

Comment:
```suggestion
public struct InstanceNorm2D<Scalar: TensorFlowFloatingPoint>: Layer {
```
`2d` -> `2D`. Same for other occurrences of `nD`.

Reply:
Done